### DISCUSSION TITLE
prompt with multilingual blending for Chinese

### DISCUSSION THREAD

**Discussion by metercai** [lllyasviel/Fooocus#908] [Ideas]
the prompt text can be a mixture of Chinese and English, more suitable for Chinese user : https://github.com/metercai/SimpleSDXL

**Comment by xhoxye** [comment]
你好，请问哪一个是需要下载的模型文件？下载后需要改名吗？存放到指定文件夹？
我下载了整个文件夹，出现这个错误，并不进行翻译
![QQ截图20231117070837](https://github.com/lllyasviel/Fooocus/assets/129571231/05fb6fb7-c762-439b-a665-1bc4a27edf53)
![QQ截图20231117070726](https://github.com/lllyasviel/Fooocus/assets/129571231/f1fd546b-695c-4108-bcb8-fcd2693b5d4c)

**Comment by xhoxye** [comment]
在我将config.txt改名为config.json后，终于可以正常翻译了
![QQ截图20231117073551](https://github.com/lllyasviel/Fooocus/assets/129571231/10875bd4-f785-4949-a777-f62dcf23c3c9)
经过了几次错误才正常进行
1、无法正确生成fooocus的配置文件，我是直接复制另一个fooocus原版的配置文件才正常进行的
![QQ截图20231117074027](https://github.com/lllyasviel/Fooocus/assets/129571231/993db6d9-e766-42c0-b739-5883a4268b37)
2、下载翻译模型说明不够详细，一开始不知道要直接下载整个文件夹
3、翻译文件夹里的config.txt不能自动改名为config.json，我手动修改

另外提一些建议：
翻译尽量放在cn,json语言文档，以减少更新时的工作量
自己写的代码尽量在新建文件，以减少更新时的作量

**Reply by metercai** [reply]
感谢使用，回复如下：
1，没在windows环境下测试过，不太熟悉。在linux环境下和运行原版fooocus是一样的，很流畅。找机会在Windows上测试修订一下。
2，翻译模型放置的路径解释的不够清楚，后续增补一下文档。
3，原版fooocus的多语言机制比较简单，满足不了界面翻译的需求，所以采用了外挂的翻译替换脚本。目前已经连续升级五六个版本，与原版完全同步，只需很少的工作量。比如最近的faceswap功能、config配置等变更都是第一时间平移的。
4，后续有计划在算力云化及界面定制化上做些创新，fooocus原作者不一定关注，只能单拉一条线往前走了。欢迎大家持续关注。

再次感谢！

**Comment by robinwang25** [comment]
要如何使用，我下载了整个压缩包后替换Fooocus下的文件无法启动

**Reply by metercai** [reply]
参照Fooocus制作了Windows的一键启动包，可以下载试试： https://github.com/metercai/SimpleSDXL

**Comment by xhoxye** [comment]
如果图片太多，缩略图显示不了。生成新图片后，不显示新图片，直接跳回显示老旧的图片合集
![QQ截图20231118211300](https://github.com/lllyasviel/Fooocus/assets/129571231/bc4809f1-82fc-4085-be75-9286814bc95e)

**Reply by metercai** [reply]
已更新，将过多图片的目录分页，修复不显示新图片的bug。
